gulp/tasks/styles.js

```diff
@@ -76,13 +77,12 @@ function stylesIE8() {
     .pipe( gulpChanged( configStyles.dest, { extension: '.ie8.css' } ) )
     .pipe( gulpLess( configStyles.settings ) )
     .on( 'error', handleErrors )
-    .pipe( gulpAutoprefixer( {
-      browsers: BROWSER_LIST.ONLY_IE_8
-    } ) )
-    .pipe( mqr( {
-      width: '75em'
-    } ) )
-    // mqr expands the minified file
+    .pipe( gulpPostcss( [
+        postcssUnmq( {
```

Comment:
Shouldn't this have been in another PR?

Reply:
See note. `css-parse`, which is a dependency of `gulp-mq-remove`, couldn't handle https://github.com/cfpb/capital-framework/blob/canary/src/cf-typography/src/licensed-fonts.less for some reason. Since it hasn't been updated in three years my guess is there is some newer syntax less outputs that it doesn't know what to do with. You can replicate this by bumping `cf-typography` to the latest on `master` and running `./frontend.sh production`.

Reply:
🤦‍♂️ Sorry, I should have paid more attention to the opening comment.